btcpay-restore.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+if [ "$(id -u)" != "0" ]; then
+  echo "This script must be run as root."
+  echo "Use the command 'sudo su -' (include the trailing hypen) and try again"
+  exit 1
+fi
+
+backup_file=$1
+if [ -z "$backup_file" ]; then
+  echo "Usage: btcpay-restore.sh /path/to/backup.tar.gz"
+  exit
+fi
+
+if [ ! -f "$backup_file" ]; then
+  echo "$backup_file does not exist."
+  exit
```

Comment:
use `exit 1` since it didn't really restore anything.

Reply:
Done in [0444ff3](https://github.com/btcpayserver/btcpayserver-docker/pull/641/commits/0444ff3e69c703dc4e1e3c767135d337c15af66c).